source/index.rst

```diff
@@ -11,7 +11,184 @@
+
+   <hr>
+
+.. panels::
+   :column: col-sm-6 d-flex pb-3
+   :footer: bg-light
+
+   ---
+   :header: font-weight-bold text-dark
+
+   Hardware Overview
+
+   ^^^
+
+   An overview of the hardware components available for teams.
+
+   +++
+
+   .. link-button:: /docs/controls-overviews/control-system-hardware
+      :type: ref
+      :text: Go to Hardware Overview
+      :classes: btn-outline-primary btn-block
+
+   ---
+   :header: font-weight-bold text-dark
+
+   Software Overview
+
+   ^^^
+
+   An overview of the software components and tools available for teams.
+
+   +++
+
+   .. link-button:: /docs/controls-overviews/control-system-software
+      :type: ref
+      :text: Go to Software Overview
+      :classes: btn-outline-primary btn-block
+
+.. raw:: html
+
+   <hr>
+
+.. panels::
+   :header: text-black font-weight-bold
+   :column: col-sm-4 d-flex pb-3
+   :footer: bg-light
+
+   VS Code Basics
+
+   ^^^
+
+   VS Code is the official Integrated Development Environment for FRC\ |reg|. These documents highlight the usage necessary for teams.
+
+   +++
+
+   .. link-button:: /docs/software/vscode-overview/index
+      :type: ref
+      :text: Go to Page
+      :classes: btn-outline-primary btn-block
+
+   ---
+
+   FRC Driver Station
+
+   ^^^
+
+   FRC Driver Station provided by NI is the tool used for enabling and controlling your robot.
+
+   +++
+
+   .. link-button:: /docs/software/driverstation/index
+      :type: ref
+      :text: Go to Page
+      :classes: btn-outline-primary btn-block
+
+   ---
+
+   Hardware Basics
+
+   ^^^
+
+   Get introduced into the FRC Control System Components and hardware for building your robot.
+
+   +++
+
+   .. link-button:: /docs/hardware/hardware-basics/index
+      :type: ref
+      :text: Go to Page
+      :classes: btn-outline-primary btn-block
+
+   ---
+
+   roboRIO Information
+
+   ^^^
+
+   roboRIO provided by NI is the robot processor responsible for directly controlling your robot.
+
+   +++
+
+   .. link-button:: /docs/software/roborio-info/index
+      :type: ref
+      :text: Go to Page
+      :classes: btn-outline-primary btn-block
+
+   ---
+
+   Command-Based
+
+   ^^^
+
+   Command-Based programming is a programming framework that helps provide structure through subsytems and commands.
+
+   +++
+
+   .. link-button:: /docs/software/commandbased/index
+      :type: ref
+      :text: Go to Page
+      :classes: btn-outline-primary btn-block
+
+   ---
+
+   Dashboards
+
+   ^^^
+
+   Dashboards such as Shuffleboard and Glass are integral means of viewing diagnostic and other informative data from your robot.
+
+   +++
+
+   .. link-button:: /docs/software/dashboards/index
+      :type: ref
+      :text: Go to Page
+      :classes: btn-outline-primary btn-block
+
+Not all content on the website is listed in the above panels. Please see the below sections for the full list of available content. Alternatively, you can use the search bar in the top-left to find content.
```

Comment:
Do we need the TOC now, or can we just refer people to the TOC on the left?

Reply:
We can't easily hide the ToC from the body without messing up navigation.

Reply:
It looks like there is a :hidden: directive that can be used to do this. https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#toctree-directive